mass_mailing_custom_unsubscribe/static/src/js/contact.tour.js

```diff
@@ -80,16 +77,15 @@ odoo.define("mass_mailing_custom_unsubscribe.contact_tour", function (require) {
                 content: "Subscribe again to list 0",
                 // eslint-disable-next-line no-multi-str
                 trigger:
-                    "body:not(:has(#unsubscribe_form #custom_div_feedback\
-                            :visible)):has(.alert-success) li:contains('test list 0') \
-                            input:not(:checked)",
+                    "body:not(:has(#unsubscribe_form #custom_div_feedback" +
+                    ":visible)):has(.alert-success) li:contains('test list 0') " +
```

Comment:
Are you sure this change is OK?

- `#custom_div_feedback :visible` means "find anything visible inside an element with ID custom_div_feedback
- `#custom_div_feedback:visible` means "find any visible element with ID custom_div_feedback

Maybe it was wrong before, but in any case it seems like the typical poisoned change. Please double-check it.

Reply:
well, I left it as it was in v13 and I checked it again and it seems to me that it is right as it is.